pkgs/servers/gemini/molly-brown/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildGoPackage , fetchgit, nixosTests }:
+
+buildGoPackage  rec {
+  name = "molly-brown-unstable-${version}";
+  version = "2020-07-06";
```

Comment:
```suggestion
  pname = "molly-brown";
  version = "unstable-2020-07-06";
```

Reply:
Fixed locally, and I've made a PR to go2nix.